src/components/DriverCard.jsx

```diff
@@ -0,0 +1,36 @@
+function DriverCard(props){
+    const {img, name, rating, car} = props;
+    
+
+    function driverRating(){
+        if (rating <= 0.49) {
```

Comment:
Here you should have used your previous `Rating.jsx` component instead of repeating all the code again.